lib/Sema/CSSimplify.cpp

```diff
@@ -1208,6 +1208,17 @@ static bool allowsBridgingFromObjC(TypeChecker &tc, DeclContext *dc,
   return true;
 }
 
+/// Determine whether the given type variables occurs in the given type.
+static bool typeVarOccursInType(TypeVariableType *typeVar, Type type) {
```

Comment:
The solver also has a copy of this but uses the CS to get a representative for the equivalence class of tyvars.  Do you think it should be factored out?